extension.js

```diff
@@ -421,18 +443,108 @@ class MprisLabel extends PanelMenu.Button {
 		return Clutter.EVENT_PROPAGATE;
 	}
 
-	_disable(){
-		if(this.icon)
-			this.box.remove_child(this.icon);
+	_disable () {
```

Comment:
Not a fan of this. Semicolons although optional in javascript are always recommended because of parser/interpreter issues (i think). After a `return` adding or not adding a semicolon shouldn't have any effect.

In any case, I'm only somewhat following the style of the [original extension](https://github.com/mheine/gnome-shell-spotify-label/blob/master/extension.js), and I don't enforce strict rules, although consistency is **pretty** *and* [pretty important](https://gjs.guide/extensions/review-guidelines/review-guidelines.html#basics)